packages/seedless-onboarding-controller/src/SeedlessOnboardingController.test.ts

```diff
@@ -179,7 +199,10 @@ async function withController<ReturnValue>(
     newRefreshToken: 'newRefreshToken',
   });
 
-  const controller = new SeedlessOnboardingController({
+  const controller = new SeedlessOnboardingController<
+    EncryptionKey | webcrypto.CryptoKey,
+    KeyDerivationOptions
+  >({
```

Comment:
I'm surprised the type parameters did not get inferred here 🤔 Should be "inferable" from `encryptor` no?

Reply:
It usually works with functions, but TypeScript does not infer type parameters depending on class constructors unless it's the generic type is explicitly specified when instantiating the object 

So we need to specify the derivation options type in the same way we specify the key type

Reply:
Yes, but given that we were not providing the `EncryptionKey` type parameter explicitly before this PR, I thought it would have worked the same with a 2nd parameter 😅 anyway, I was just curious! Thanks.

Reply:
On this note, I'm wondering if we should add the same to `KeyringController`.. thoughts?